src/main/target/FLYWOOF411EVO_HD/target.h

```diff
@@ -0,0 +1,149 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define USE_TARGET_CONFIG
+
+#define TARGET_BOARD_IDENTIFIER "FLWO"
+#define USBD_PRODUCT_STRING     "FLYWOOF411EVO_HD"
+
+#define LED0_PIN                PC13
+
+#define USE_BEEPER
+#define BEEPER_PIN              PC14
+#define BEEPER_INVERTED
+
+// *************** SPI **********************
+#define USE_SPI
+#define USE_SPI_DEVICE_1
+#define SPI1_SCK_PIN               PA5
+#define SPI1_MISO_PIN              PA6
+#define SPI1_MOSI_PIN          PA7
+
+#define USE_SPI_DEVICE_2
+#define SPI2_SCK_PIN               PB13
+#define SPI2_MISO_PIN          PB14
+#define SPI2_MOSI_PIN          PB15
+
+// *************** Gyro & ACC **********************
+#define MPU6000_CS_PIN          PA4
+#define ICM20689_CS_PIN         PA4
```

Comment:
I don't think ICM20689 definitions are required for this target. Pretty sure its MPU6000 only, but don't want them removed without testing and the only tester says it flies great now.